appveyor.yml

```diff
@@ -0,0 +1,45 @@
+environment:
+
+  global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci\\appveyor\\run_with_env.cmd"
+
+  matrix:
+#    - PYTHON: "C:\\Python27_32"
+#      PYTHON_VERSION: "2.7"
+#      PYTHON_ARCH: "32"
+#      CONDA_PY: "27"
+#      CONDA_NPY: "18"
+
+    - PYTHON: "C:\\Python27_64"
+      PYTHON_VERSION: "2.7"
+      PYTHON_ARCH: "64"
+      CONDA_PY: "27"
+      CONDA_NPY: "18"
+
+#    - PYTHON: "C:\\Python34_32"
+#      PYTHON_VERSION: "3.4"
+#      PYTHON_ARCH: "32"
+#      CONDA_PY: "34"
+#      CONDA_NPY: "18"
+
+#    - PYTHON: "C:\\Python34_64"
+#      PYTHON_VERSION: "3.4"
+#      PYTHON_ARCH: "64"
+#      CONDA_PY: "34"
+#      CONDA_NPY: "18"
+
+install:
+  # this installs the appropriate Miniconda (Py2/Py3, 32/64 bit),
+  # as well as pip, conda-build, and the binstar CLI
+  - powershell .\\ci\\appveyor\\install.ps1
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+
+build: false
+
+test_script:
+  - "%CMD_IN_ENV% conda install --yes hdf5 numexpr cython"
```

Comment:
great work @joshmoore, maybe add [`--quiet`](http://conda.pydata.org/docs/commands/conda-install.html) so the the progress bar doesn't litter the log?